docs: Add simplified installation instructions for lint-staged (#5911)
prettier/prettier: docs/precommit.md

## Option 1. [lint-staged](https://github.com/okonet/lint-staged)

> **Use Case:** Useful for when you want to use other code quality tools along with Prettier (e.g. ESLint, Stylelint, etc.). Supports partially staged files (`git add --patch`)

Reviewer: If someone needs the partially staged files, they will read the docs.. maybe it's too much info for this page.

Author: According to lint-staged feedback, this was one of the most requested features. That's why I wanted to make it more prominent since apparently people are looking for this. I've updated the sentence a bit to make it clear, though. WYT?

_Make sure Prettier is installed and is in your `devDependencies` before you proceed._

```bash
npx mrm lint-staged
```

> This will install and configure [husky](https://github.com/typicode/husky) and [lint-staged](https://github.com/okonet/lint-staged) to reformat supported formats on pre-commit hook.

Reviewer: Shall we mention that the `package.json` will be updated with the necessary rules and if users needs to support more files they should edit there? (Something among those lines, that the pre-commit hooks are done in package.json and not under the .git directory for example)

Author: Updated the wording. WYT?

> This will install [husky](https://github.com/typicode/husky) and [lint-staged](https://github.com/okonet/lint-staged) and add a configuration for automatic formatting of supported files as a pre-commit hook to project's `package.json`.

Reviewer 2: ```suggestion
This will install [husky](https://github.com/typicode/husky) and [lint-staged](https://github.com/okonet/lint-staged), then add a configuration to the project’s `package.json` that will automatically format supported files in a pre-commit hook.
```
